time stamp

bool[4] selected
float64[4] trust